app/main/views/questionnaire.py

```diff
@@ -30,11 +36,11 @@
 @questionnaire_blueprint.before_request
 @login_required
 def check_survey_state():
-    json, schema = get_schema()
-    g.questionnaire_manager = QuestionnaireManager(schema, json=json)
+    g.schema_json, g.schema = get_schema()
```

Comment:
Have we determined a way to bind at an application level, rather than per request in order to cache the schema?